src/Analysis/Engine/Test/AnalysisTest.cs

```diff
@@ -3132,23 +3132,45 @@ not abc
         print(a)
         assert isinstance(a, int)
         a = 200
+        print(a)
 ";
                 await server.SendDidOpenTextDocument(uri, text);
 
-                var references = await server.SendFindReferences(uri, 3, 15);
-                references.Should().OnlyHaveReferences(
+                var expected = new (Uri documentUri, (int, int, int, int), ReferenceKind?) [] {
```

Comment:
You don't need `documentUri` here: `var expected = new (Uri, (int, int, int, int), ReferenceKind?) [] {`

Reply:
Or maybe we should just add an overload for `OnlyHaveReferences` with `(Uri documentUri, (int, int, int, int), ReferenceKind)` tuple.